daemon/api_test.go

```diff
@@ -1638,10 +1638,11 @@ func (s *apiSuite) TestInstallFails(c *check.C) {
 	c.Check(chg.Err(), check.ErrorMatches, `(?sm).*Install task \(fake-install-snap-error errored\)`)
 }
 
+/* temporarely not available
```

Comment:
you could instead `c.Skip("temporarily not available")` which tends to not let the code bitrot too much. Also, temporarily not temporarely.